srcpkgs/mftrace/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'mftrace'
+pkgname=mftrace
+version=1.2.19
+revision=1
+build_style=gnu-configure
+depends="python potrace"
```

Comment:
python2 only? :confused:

Reply:
The main executable explicitly uses `/usr/bin/python2`, yes...